app/helpers/mentions_helper.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module MentionsHelper
+  def mention_matcher
```

Comment:
private method ?

Reply:
Modules don't have private methods. Which makes me want to check how that works again 🤔
But I verified, by adding `private` it does not prevent the view context from accessing the method. I think everything in a helper is fair game to the view.